Dockerfile.profiles

```diff
@@ -0,0 +1,10 @@
+# Go build
```

Comment:
We weren't sure where folks would want the Dockerfile to live. How would folks feel about us creating a `build/` directory, containing the two docker files? Or any other suggestions?

Reply:
moving it to `cmd/profiles` sounds good. If its in there I guess just `Dockerfile` would be okay?

Also how would you feel if we also moved the top level `Dockerfile` into `cmd/gitops`, just for consistency?

Reply:
https://github.com/weaveworks/weave-gitops/pull/1148/commits/01b38b1b0e5cae11d9d9c2b3a37c62a95942711b

Reply:
as per discussions in slack I'm going to remove this dockerimage and move the profiles API into the app api